website/static/js/registrationUtils.js

```diff
@@ -32,59 +32,60 @@ var currentUser = window.contextVars.currentUser || {
  * @property {String} name
  **/
 function Comment(data) {
-    var self = this;
+        var self = this;
 
-    self.saved = ko.observable(data ? true : false);
+        self.saved = ko.observable(data ? true : false);
 
-    data = data || {};
-    self.user = data.user || currentUser;
-    self.lastModified = new Date(data.lastModified)|| new Date();
-    self.value = ko.observable(data.value || '');
-    self.value.subscribe(function() {
-        self.lastModified = new Date();
-    });
+        data = data || {};
+        self.user = data.user || currentUser;
+        self.lastModified = new Date(data.lastModified) || new Date();
+        self.value = ko.observable(data.value || '');
+        self.value.subscribe(function() {
+            self.lastModified = new Date();
+        });
 
-    self.isDeleted = ko.observable(data.isDeleted || false);
-    self.isDeleted.subscribe(function(isDeleted) {
-      if (isDeleted) {
-        self.value('');
-      }
-    });
+        self.created = new Date(data.created) || new Date();
 
-    self.seenBy = ko.observableArray([self.user.id] || []);
-    /**
-     * Returns the author as the actual user, not 'You'
-     **/
-    self.author = ko.pureComputed(function() {
-      return self.user.fullname;
-    });
+        self.isDeleted = ko.observable(data.isDeleted || false);
+        self.isDeleted.subscribe(function(isDeleted) {
+            if (isDeleted) {
+                self.value('');
+            }
+        });
 
-    /**
-     * Returns 'You' if the current user is the commenter, else the commenter's name
-     */
-    self.getAuthor = ko.pureComputed(function() {
-      if (self.user.id === currentUser.id) {
-        return 'You';
-      }
-      else {
-        return self.user.fullname;
-      }
-    });
+        self.seenBy = ko.observableArray([self.user.id] || []);
+        /**
+         * Returns the author as the actual user, not 'You'
+         **/
+        self.author = ko.pureComputed(function() {
+            return self.user.fullname;
```

Comment:
@HarryRybacki this shouldn't be possible. A user will have to be a contributor on the project and logged in to comment or to access the form.